packages/apptive_grid_form/lib/widgets/form_widget/user/user_dropdown_button_form_field.dart

```diff
@@ -0,0 +1,254 @@
+part of apptive_grid_form_widgets;
+
+class _UserDropdownButtonFormField extends StatefulWidget {
+  const _UserDropdownButtonFormField({
+    required this.component,
+    required this.onSelected,
+  });
+
+  final FormComponent<UserDataEntity> component;
+
+  final void Function(DataUser? user) onSelected;
+
+  @override
+  _UserDropdownButtonFormFieldState createState() =>
+      _UserDropdownButtonFormFieldState();
+}
+
+class _UserDropdownButtonFormFieldState
+    extends State<_UserDropdownButtonFormField>
+    with AutomaticKeepAliveClientMixin {
+  @override
+  bool get wantKeepAlive => true;
+
+  final _filterController = TextEditingController();
+
+  final _overlayKey = GlobalKey();
+
+  void closeOverlay() {
+    if (_overlayKey.currentContext != null) {
+      Navigator.pop(_overlayKey.currentContext!);
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    super.build(context);
+    return DropdownButtonFormField<DataUser>(
```

Comment:
I'm a bit confused on why this is an always expanded dropdown from with only ever 2 items. Is there a reason, why it can't just be a column with those 2 elements?

Reply:
Ok, that makes sense. The drop down name confused me, but after visualising it, it does make sense like this.